pkg/compute/publish.go

```diff
@@ -17,8 +17,11 @@ type PublishCommand struct {
 	deploy   *DeployCommand
 
 	// Deploy fields
-	path    common.OptionalString
-	version common.OptionalInt
+	path        common.OptionalString
```

Comment:
11. Now `init` and `deploy` have been updated we need to ensure `publish` is kept in-sync as it proxies through to the `deploy` subcommand.